.travis.yml

```diff
@@ -23,8 +23,8 @@ env:
 
 jobs:
   include:
-    - stage: lint
-      name: "Tox tests and coverage"
+    - &linting
```

Comment:
Here we apply the same technique that we use in the `test` stage, explained [here](https://github.com/kivy/python-for-android/pull/1835#discussion_r291112884)